src/apps/dashboard/routes/notifications.tsx

```diff
@@ -1,36 +1,34 @@
+import Alert from '@mui/material/Alert/Alert';
+import Box from '@mui/material/Box/Box';
+import Button from '@mui/material/Button/Button';
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 import Page from 'components/Page';
 import globalize from 'scripts/globalize';
 
-const PluginLink = () => (
-    <div
-        dangerouslySetInnerHTML={{
-            __html: `<a
-                is='emby-linkbutton'
-                class='button-link'
-                href='#/dashboard/plugins/add?name=Webhook&guid=71552a5a5c5c4350a2aeebe451a30173'
-            >
-                ${globalize.translate('GetThePlugin')}
-            </a>`
-        }}
-    />
-);
-
-const Notifications = () => (
+const NotificationsPage = () => (
     <Page
         id='notificationSettingPage'
         title={globalize.translate('Notifications')}
         className='mainAnimatedPage type-interior'
     >
         <div className='content-primary'>
             <h2>{globalize.translate('Notifications')}</h2>
-            <p>
-                {globalize.translate('NotificationsMovedMessage')}
-            </p>
-            <PluginLink />
+
+            <Alert severity='info'>
+                <Box sx={{ marginBottom: 2 }}>
+                    {globalize.translate('NotificationsMovedMessage')}
+                </Box>
+                <Button
+                    component={Link}
+                    to='/dashboard/plugins/add?name=Webhook&guid=71552a5a5c5c4350a2aeebe451a30173'
```

Comment:
Slightly unrelated, when you don't have a repository that includes this specific plugin the page will fail to load. We should probably show some error on it.

Reply:
Yeah definitely for a separate PR, just wanted to let you know